TODO.md

```diff
@@ -10,3 +11,12 @@ Designate Zones:
 
 select tenant_id, COUNT(id) as total_zones from zones WHERE tenant_id != '00000000-0000-0000-0000-000000000000' GROUP BY t
 enant_id;
+
+Nova local storage:
+
+SELECT project_id, SUM(vcpus) AS total_vcpus, SUM(memory_mb) AS total_ram_mb, SUM(root_gb) as total_root_gb from instances WHERE deleted = 0 GROUP BY project_id;
+
+Cinder / Nova Snapshot / Backup Storage
+Octavia (number of LBs)
+Swift/S3 (Storage in GiB)
```

Comment:
Swift/S3 is out of this scope as it is not possible to get this info (in our case) from the DB. see https://github.com/scaleup-technologies/rgw-bucket-exporter